modules/learn/pages/clusters-and-availability/rebalance.adoc

```diff
@@ -150,14 +151,35 @@ The rebalance process takes into account the nodes' CPU, RAM, and disk bandwidth
 By default, Couchbase Server rebuilds the indexes in their new locations during a rebalance.  
 If your cluster uses xref:learn:services-and-indexes/indexes/storage-modes.adoc#standard-index-storage[Standard Index Storage], you can choose to enable file-based rebalance which moves index files between nodes instead of rebuilding them. 
 Copying the index files is faster than having the target node rebuild the index from scratch.
-However, if you enable file-based rebalancing, you lose the ability to choose which Index Service node handles a specific index.  
+You cannot use file-based rebalance if you have enabled xref:learn:services-and-indexes/indexes/storage-modes.adoc#memory-optimized-index-storage[Memory Optimized Index Storage] because it does not store index data in files.
```

Comment:
> it does not store index data in files
it does not support grouping indexes which is a pre-requisite for file based rebalance